packs/spell-effects/spell-effect-element-embodied-metal.json

```diff
@@ -0,0 +1,118 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 10
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": true,
+            "title": "Pathfinder Rage of Elements"
+        },
+        "rules": [
+            {
+                "hasHands": true,
+                "key": "BattleForm",
+                "overrides": {
+                    "armorClass": {
+                        "modifier": "25 + @actor.level"
+                    },
+                    "resistances": [
+                        {
+                            "type": "electricity",
+                            "value": 10
+                        }
+                    ],
+                    "senses": {
+                        "darkvision": {}
+                    },
+                    "size": "grg",
+                    "skills": {
+                        "acr": {
```

Comment:
The Acrobatics/Athletics for metal and wood weren't listed in the PDF, so I went with the skill matching their strike attribute.